test/NuGet.Tests.Apex/NuGet.OptProf/Assets/PackageReferenceSdk/Directory.Packages.props

```diff
@@ -1 +1,5 @@
-<Project />
```

Comment:
If you run the test from VS, or `dotnet test` on the CLI, I'm worried it might find the repo root Directory.Packages.props. I assume that's why the file was added in the first place. There are also empty Directory.Build.props and targets files in the same directory.